docs/source/tutorials/memory_optimizations.rst

```diff
@@ -8,14 +8,29 @@ Memory Optimization Overview
 
 torchtune comes with a host of plug-and-play memory optimization components which give you lots of flexibility
 to ``tune`` our recipes to your hardware. This page provides a brief glossary of these components and how you might use them.
+To make things easy, we've summarized these components in the following table:
+
+.. csv-table:: Memory optimization components
```

Comment:
I like the table, but i think it misses the trade-offs/numbers and for the most part repeats: "Use when memory constrained". Maybe an index would be fine?

When we have the table with all the % value of VRAM/tps, i think we could put it back here. WDYT?

Reply:
I've tried to callout a bit more the compromises each component requires on training speed, does that help? I'd like to make this useful and help users who don't want to read the whole tutorial, so very open to suggestions.